apps/src/Globals.d.ts

```diff
@@ -32,6 +32,11 @@ declare module '*.svg' {
   export = value;
 }
 
+declare module '*.gif' {
```

Comment:
Minor unrelated change: needed for TS to handle the .gif import, which we're now using for the DancePartyLoading GIF.